src/system_info/system_info_network_desktop.cc

```diff
@@ -2,10 +2,12 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#include "system_info/system_info_network.h"
+#include "system_info/system_info_network_desktop.h"
 
 #include <NetworkManager.h>
 
+const std::string SysInfoNetwork::name_ = "NETWORK";
```

Comment:
This part should be common for all platforms. How about keep system_info_network.cc available to host those common things.